### ISSUE TITLE
Quoted unpack operator reported as error

### ISSUE THREAD

**Issue by BlueGlassBlock** [microsoft/pyright#3768]
**Describe the bug**
Unpack operators wrapped in **quoted annotation** are reported as errors.

**To Reproduce**
```python
from __future__ import annotations

from typing import TypeVar

from typing_extensions import TypeVarTuple, Unpack

TVT_A = TypeVarTuple("TVT_A")
T = TypeVar("T")


def append(tup1: "tuple[*TVT_A]", obj: T) -> "tuple[Unpack[TVT_A], T]":
    #                   ^                                  ^^^^^
    # Unpack operator in subscript requires Python 3.11 or newer
    # TypeVar "TVT_A" appears only once in generic function signature
    return tup1 + (obj,) # for demonstration only
```

**Expected behavior**
The code is considered to be correct by pyright.

**Screenshots or Code**
Code: see above.
![image](https://user-images.githubusercontent.com/50769752/182011979-71755c47-ace6-409c-9582-e85975838e47.png)

**VS Code extension or command-line**
Pylance v2022.7.43

**Additional context**

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.265, which I just published. It will be included in a future release of pylance.
